src/Filter/Filter.php

```diff
@@ -24,6 +25,15 @@ abstract class Filter extends BaseFilter
      */
     protected $active = false;
 
+    /**
+     * Holds an array of `orX` expressions used by each admin when the condition
+     * equals the value on `FilterInterface::CONDITION_OR`, using the admin code
+     * as index.
+     *
+     * @var array<string, Orx>
+     */
+    private static $orExpressionsByAdmin = [];
```

Comment:
This property (and the "admin_code" option) was renamed in #1373. Could you please perform the merge again?